Sources/armory/renderpath/Inc.hx

```diff
@@ -571,7 +571,7 @@ class Inc {
 			t.height = 0;
 			t.displayp = getDisplayp();
 			t.scale = getSuperSampling();
-			t.format = getHdrFormat();
+			t.format = "RGBA32";
```

Comment:
This can' t be set to RGBA64 and is not meant to

Reply:
Why is `voxels_ao` set to `RGBA32` instead of `R8` though?

Reply:
@luboslenco I was reworking the whole stuff and did this for debugging purpose, to see a proper black and white image of the AO, I am going to roll that back.